doc/connectivity/networking/api/wifi.rst

```diff
@@ -24,6 +24,43 @@ Currently, two types of Wi-Fi drivers are supported:
 * Networking or socket offloaded drivers
 * Native L2 Ethernet drivers
 
+Wi-Fi Enterprise test: X.509 Certificate header generation
+##########################################################
+
+Wi-Fi enterprise security requires use of X.509 certificates, test certificates
+in PEM format are committed to the repo and the below steps to be followed to convert
+them to C header file that can be included in Wi-Fi shell module.
+
+Convert CA format from PEM to DER
+=======================================
+To convert the CA format from PEM to DER, use the following command:
+.. code-block:: none
```

Comment:
This isn't being rendered properly, needs a newline before as well.

Reply:
Hi @krish2718 thanks, please review again

Reply:
FYI, when you do a force push after resolving comments, reviewers get an email, and you can still trigger a specific email for a reviewer by clicking on `refresh` button as below.

![image](https://github.com/user-attachments/assets/425bdc8e-0ec6-4796-9d31-dde95ec19508)

Reply:
thanks for your info